pkgs/development/python-modules/beancount-parser/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, isPy3k
+, lark
+, poetry
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "beancount-parser";
+  version = "0.1.19";
+
+  disabled = !isPy3k;
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "LaunchPlatform";
+    repo = "beancount-parser";
+    rev = version;
+    sha256 = "sha256-59Fc9zfSkmCBRTeqB6VTTSi2f/HNwKHihVvk9sEuZ3A=";
+  };
+
+  buildInputs = [
+    poetry
```

`poetry-core` should be [sufficient](https://github.com/LaunchPlatform/beancount-parser/blob/master/pyproject.toml#L18).

```suggestion
    poetry-core
```